pkg/pool-manager/pool_test.go

```diff
@@ -414,4 +416,75 @@ var _ = Describe("Pool", func() {
 		})
 	})
 
-})
+	Describe("Multus Network Annotations API Tests", func() {
+		BeforeEach(func() {
+			poolManager := createPoolManager("02:00:00:00:00:00", "02:00:00:00:00:01", &vmConfigMap)
+			Expect(poolManager).ToNot(Equal(nil), "should create pool-manager")
+
+			Context("When pool-manager is configured with available addresses", func(){
```

Comment:
For the `"When`. I think it is better to start with either lower case (so the test name is readable as one long sentence) or with all capitals `WHEN` (so it is CLEAR in the test name that the context starts there)

Reply:
ack